src/components/PhoneAutocomplete/index.js

```diff
@@ -47,6 +48,61 @@ export const PhoneAutocomplete = (props) => {
       setCustomersPhones({ ...customersPhones, loading: false, error: e.message })
     }
   }
+  /**
+   * fetch business to get its address
+   */
+  const getBusiness = async () => {
+    const { content: { result, error } } = await ordering
+      .businesses('mcbonalds')
+      .select([
+        'address',
+        'location',
+        'distance'
+      ])
+      .parameters()
+      .get()
+    if (error) {
+      setAlertState({ open: true, content: result })
+      return
+    }
+    setBusinessAddress(result)
+  }
+
+  const setBusinessAddressToUser = async (userId, onRedirect) => {
+    if (!businessAddress) return
+    try {
+      const { content: { result: resultAddresses, error } } = await ordering.users(userId).addresses().get()
+      if (error) {
+        setAlertState({ open: true, content: resultAddresses })
+        return
+      }
+      const userBusinessAddress = resultAddresses.find((address) => address.address === businessAddress.address || address.location === businessAddress.location)
+      let addressId = userBusinessAddress?.id
+      if (!userBusinessAddress) {
+        const response = await ordering.users(userId).addresses().save({ address: businessAddress.address, location: businessAddress.location })
+        if (response.content.error) {
+          setAlertState({ open: true, content: response.content.result })
+          return
+        }
+        addressId = response.content.result.id
+      }
+      const addressResponse = await ordering.users(userId).addresses(addressId).save({ default: true })
+      if (addressResponse.content.error) {
+        setAlertState({ open: true, content: addressResponse.content.result })
```

Comment:
You only need pass the state, you don't need use any UI component for ordering-components lib. This is a old requirement